src/services/vuestic-ui/components.js

```diff
@@ -0,0 +1,9 @@
+import VuesticPlugin from 'vuestic-ui/src/components/vuestic-plugin'
```

Comment:
There is not much of a point grouping these instead of importing them directly.
But that's a minor thing.

Reply:
One point has a value if we will do refactor in the future. And it's better for understanding.